e2e/desktop/tests/specs/swap.spec.ts

```diff
@@ -132,8 +143,22 @@ for (const { swap, xrayTicket } of swaps) {
     });
 
     test.use({
-      userdata: "speculos-tests-app",
+      userdata: "skip-onboarding",
       speculosApp: app,
+
+      cliCommandsOnApp: [
+        [
+          {
+            app: accountToDebit.currency.speculosApp,
+            cmd: liveDataCommand(accountToDebit.currency.speculosApp, accountToDebit.index),
+          },
+          {
+            app: accountToCredit.currency.speculosApp,
+            cmd: liveDataCommand(accountToCredit.currency.speculosApp, accountToCredit.index),
+          },
+        ],
+        { scope: "test" },
+      ],
     });
```

Comment:
test.use seems similar for all tests, why not factorize it all instead of doing it only for cmd ?